tests/Assetic/Test/Extension/Twig/TwigResourceTest.php

```diff
@@ -24,9 +24,9 @@ protected function setUp()
 
     public function testInvalidTemplateNameGetContent()
     {
-        $loader = $this->getMock('Twig_LoaderInterface');
+        $loader = $this->getMock('Twig_LoaderInterface', array('getSource', 'getSourceContext', 'getCacheKey', 'isFresh', 'exists'));
```

Comment:
this is wrong (you needed it because of the fact you are trying to mock a method which does not belong to Twig_LoaderInterface, showing you that you have a bug in your code)